pkg/cmd/template/schema_test.go

```diff
@@ -2533,6 +2743,141 @@ foo: 0
 
 		assertFails(t, filesToProcess, expectedErr, opts)
 	})
+	t.Run("when schema/default annotation", func(t *testing.T) {
+		t.Run("value is empty", func(t *testing.T) {
+			schemaYAML := `#@data/values-schema
+---
+#@schema/default
+foo: 0
+`
+			templateYAML := `#@ load("@ytt:data", "data")
+---
+foo: #@ data.values.foo
+`
+			expectedErr := `
+Invalid schema
+==============
+expected @schema/default annotation to contain default value
+
+schema.yml:
+    |
+  4 | foo: 0
+    |
+
+    = found: missing value
+    = expected: integer (by schema.yml:4)
+    = hint: value must be the same type as the annotated node.
+    = hint: value must be in Starlark format, e.g.: {'key': 'value'}, True.
+`
+
+			filesToProcess := files.NewSortedFiles([]*files.File{
+				files.MustNewFileFromSource(files.NewBytesSource("schema.yml", []byte(schemaYAML))),
+				files.MustNewFileFromSource(files.NewBytesSource("template.yml", []byte(templateYAML))),
+			})
+
+			assertFails(t, filesToProcess, expectedErr, opts)
+		})
+		t.Run("has multiple values", func(t *testing.T) {
+			schemaYAML := `#@data/values-schema
+---
+#@schema/default 1, 2
+foo: 0
+`
+			templateYAML := `#@ load("@ytt:data", "data")
+---
+foo: #@ data.values.foo
+`
+			expectedErr := `
+Invalid schema
+==============
+expected @schema/default annotation to contain one argument as default value
+
+schema.yml:
+    |
+  4 | foo: 0
+    |
+
+    = found: 2 values
+    = expected: integer (by schema.yml:4)
+    = hint: value must be the same type as the annotated node.
+    = hint: value must be in Starlark format, e.g.: {'key': 'value'}, True.
```

Comment:
Similar "critique" as above. When writing hints, I'm asking myself, "what is the most likely miscommunication with the user, here? what can we succinctly add to repair that ?"

```suggestion
Invalid schema
==============
syntax error in @schema/default annotation

schema.yml:
    |
  4 | foo: 0
    |

    = found: 2 values (in @schema/default above this item)
    = expected: integer (by schema.yml:4)
```